packages/playwright-core/src/server/dispatchers/networkDispatchers.ts

```diff
@@ -39,13 +40,15 @@ export class RequestDispatcher extends Dispatcher<Request, channels.RequestChann
   private constructor(scope: DispatcherScope, request: Request) {
     const postData = request.postDataBuffer();
     super(scope, request, 'Request', {
-      frame: FrameDispatcher.from(scope, request.frame()),
+      frame: FrameDispatcher.fromNullable(scope, request.frame()),
+      serviceWorker: WorkerDispatcher.fromNullable(scope, request.serviceWorker()),
       url: request.url(),
       resourceType: request.resourceType(),
       method: request.method(),
       postData: postData === null ? undefined : postData.toString('base64'),
       headers: request.headers(),
       isNavigationRequest: request.isNavigationRequest(),
+      isServiceWorkerRequest: request.isServiceWorkerRequest(),
```

Comment:
Why send this one over the protocol? It can be easily computed on the client side.